.htaccess

```diff
@@ -22,6 +22,7 @@
     RewriteRule ^(.*)$ %{ENV:BASE}/index.php?/$1 [L,QSA]
     RewriteRule ^Uploads.* - [F]
 </IfModule>
+RedirectMatch 403 "^/(.git|.github)/.*$"
```

Comment:
Requires mod_alias, so might be better done with rewrite for consistency.

Additionally, needs to be applied to the .htaccess (apache 2.2 and 2.4) and Nginx configs in warmup/

Reply:
done